lisa/sut_orchestrator/baremetal/cluster/idrac.py

```diff
@@ -50,14 +56,47 @@ def _start(self, wait: bool = True) -> None:
             self._log.debug("System is already powered on.")
             return
         self.cluster.reset("On")
+        check_till_timeout(
```

Comment:
Add it to `reset` method, so it won't be repeated.

Reply:
update